source/blog/2016-08-20-building-compassionate-software.html.markdown

```diff
@@ -0,0 +1,143 @@
+
+This is how someone empathizes, which is core to feelings, which matter. Feelings are important. Now I want to explore what it means to be a member of a team where feelings are prioritized.
+
+## Teams with Psychological Safety Perform Better
+
+So you want to be a 10x developer, eh? You may have heard that the 10x developer is a myth, but that's not true: a 10x developer is someone who makes the five developers around them each twice as productive. You can be a 10x developer by making sure that your team has psychological safety.
+
+Psychological safety is defined as
+  
+> The belief that one will not be punished or humiliated for speaking up with ideas, questions, concerns, or mistakes.
+
+I know this sounds kind of touchy-feely, but there is data to back this up! Google spent _five years_ on [Project Aristotle](http://www.nytimes.com/2016/02/28/magazine/what-google-learned-from-its-quest-to-build-the-perfect-team.html?_r=0) searching for the answer to _what makes some teams perform really well and other teams perform poorly?_ They examined _a tonne_ of data and eventually – after extensive searching – they found psychological safety is correlated positively with team success.
+
+> There were other behaviors that seemed important as well — like making sure teams had clear goals and creating a culture of dependability. But Google’s data indicated that psychological safety, more than anything else, was critical to making a team work.
+
+This is _Google_. They A/B test shades of blue to use on the Gmail "Send" button. They are _the_ data-driven organization, and their research came to the conclusion that teams with psychological safety were more successful than teams without.
+
+Remember, psychological safety is the belief that you won't be punished for saying something. That's fairly basic, but think about it: I'm sure you've worked in groups where this wasn't the case. Was that project a success, or a failure?
+
+YOUTUBE43 LhoLuui9gX8
+
+Developers need to be able to ask questions when they don't understand something. We need to feel free to suggest ideas or concerns, to be able to point out and admit mistakes. This is really necessary for development teams, and _especially_ necessary in resource-strained startups where missteps could cost the company.
+
+There was this one time at Artsy where I was behind schedule on a big feature, which was delaying me from starting work on [something really important](http://artsy.github.io/blog/2016/08/09/the-tech-behind-live-auction-integration/). I sat down with my team, and we had _just_ started reviewing the designs for all the stuff I had to finish when a designer asked "what if... we just... didn't do any of it?" We hadn't really considered whether the feature I was behind on was worth delaying the next project. A designer from outside our team was comfortable challenging our assumptions. They were right, we dropped the delayed feature and moved directly on to what was more important.
```

Comment:
"what if... we just... didn't do any of it?" tense is off, they would have said "don't"